kpi/management/commands/update_collection_languages.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+import sys
+
+from django.core.management.base import BaseCommand
+
+from kpi.constants import ASSET_TYPE_COLLECTION
+from kpi.models.asset import Asset
+
+
+class Command(BaseCommand):
+
+    help = "Update collection languages by merging all their children's languages"
+
+    def add_arguments(self, parser):
+        super().add_arguments(parser)
+
+        parser.add_argument(
+            "--chunks",
+            default=1000,
+            type=int,
+            help="Update only records by batch of `chunks`.",
+        )
+
+    def handle(self, *args, **options):
+
+        chunks = options['chunks']
+
+        query = Asset.objects.filter(asset_type=ASSET_TYPE_COLLECTION)
+        total = query.count()
+        max_pk = 0
+        stop = False
+        cpt = 0
+
+        while not stop:
+            query = query.filter(pk__gt=max_pk).order_by('pk')
+            if query.count() == 0:
+                stop = True
+            else:
+                for collection in query.all()[:chunks]:
```

Comment:
I think now that you've upgraded Django :bow: it uses server-side cursors by default, and this type of chunking isn't needed anymore. Just a note.

Reply:
I left a comment in the code  to update it with new Django `iterator()`